config.go

```diff
@@ -141,6 +147,26 @@ func validateOptions(cfg *config) (err error) {
 		}
 	}
 
+	if cfg.vaultClientCertificate != "" {
```

Comment:
If you wish to have them grouped you could try the following. But it's no big deal.

```
if pvtk or pubk:
    if not exists(pvtk):
        raise 'unable to read/find pvtk {path}, to use client side verify a path to pvtk is required'
    if not exists(pubk):
        raise 'unable to read/find pubk {path}, to use client side verify a path to pubk is required'
```